js/lib/widget.js

```diff
@@ -244,7 +244,7 @@ var RemoteFrameBufferView = widgets.DOMWidgetView.extend({
                 if (Object.keys(that._pointers).length > 0) { return; }
             }
             let event = create_pointer_event(that.img, e, that._pointers, 'pointer_move');
-            that.send_throttled(event, 50);
+            that.send_throttled(event, 20);
```

Comment:
With 50ms the max framerate you obtain when new frames are requested on mouse move is 20 FPS. With 20ms this becomes 50 FPS.